docs/design/labelling.rst

```diff
@@ -0,0 +1,800 @@
+
+**Do:** Get started with Mautic to automate your marketing workflows and enhance customer engagement.
+
+.. vale off
+
+**Avoid:** GET STARTED WITH MAUTIC TO AUTOMATE YOUR MARKETING WORKFLOWS AND ENHANCE CUSTOMER ENGAGEMENT.
+
+.. vale on
+
+When to use capital letters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Capital letters are reserved for:
+
+- Official, trademarked products or services, unless they intentionally use a lowercase initial, such as the iPhone.
+- Initialisms (for example, FBI, CIA) or acronyms (for example, NASA, UNESCO).
+- Names of people.
+- Names of countries or places.
+- UI labels that are capitalized.
+- Words that begin a sentence or phrase.
+
+If a term is not in the list above, it should not be capitalized. Some company and product names use non-standard capitalization. When referring to these, ensure accuracy. Examples include node.js, GitHub, eBay and JavaScript.
+
+How to refer to UI elements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When writing about a UI element, use the same capitalization as seen in the UI. For example, if an input field is labeled "Email," refer to it as the Email input field. Similarly, if a button is labeled "Submit," correctly refer to it as the Submit button. If a product page is titled "User dashboard," write it as "Access your settings on the User dashboard page."
+
+**Do:** Access your settings on the User dashboard page.
+
+**Avoid:** Access your settings on the User Dashboard page.
+
+Capitalizing proper nouns
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The names of people, places, and products are proper nouns and therefore all take initial capitals. Examples of proper nouns are:
+
+.. vale off
+
+- Ruth Cheesley
+- Mautic Conference Europe 2024
+- São Paulo, Brazil
+- Microsoft Copilot
+
+.. vale on
+
+Capitalizing abbreviations
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When referencing a product name, use the official full name rather than an abbreviation or initials. Use all uppercase letters for well-recognized abbreviations, including both initialisms and acronyms, such as GIF.
+
+- ASCII
+- CAPTCHA
+- FAQ
+- HTML
+- OK (not Ok or Okay)
+
+When there's a possibility that an abbreviation might be unfamiliar to the target audience, it's advisable to spell it out in full the first time you use it. For commonly known abbreviations, such as PDF, CEO, or AM/PM, it's perfectly acceptable to use them without spelling them out.
+
+Capitalizing all other words
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Capitalize a word only if it begins a sentence, phrase, or UI element name. To emphasize a particular word or phrase within a sentence, use italic or bold formatting (but not both) instead of capitalizing it to denote "specialness."
+
+Examples:
+
+**Do:** You can use a global policy to apply changes to all Users.
+
+**Avoid:** You can use a Global Policy to apply changes to all Users
+
+
+Simple writing
```

Comment:
I think we should avoid using the word Simple - is there something else we can use instead? Perhaps 'Concise'?

Reply:
Simple is different of Concise
The goal here is to convey the opposite of complex

Reply:
Then we should use 'Clear', I think?